lib/graphql/static_validation/all_rules.rb

```diff
@@ -24,6 +24,7 @@ module StaticValidation
       GraphQL::StaticValidation::ArgumentsAreDefined,
       GraphQL::StaticValidation::ArgumentLiteralsAreCompatible,
       GraphQL::StaticValidation::RequiredArgumentsArePresent,
+      GraphQL::StaticValidation::ArgumentNamesAreUnique,
```

Comment:
I wasn't exactly sure where to add this new rule, but I think here should be fine.

Reply:
Sadly a few of them are order-dependent 😩  but I think as long as it doesn't 💥  then we're fine.